docs/guides/references/troubleshooting.mdx

```diff
@@ -247,34 +247,69 @@ an issue you have by clearing this app data.
 
 ## Print DEBUG logs
 
-Cypress is built using the [debug](https://github.com/visionmedia/debug) module.
-That means you can receive helpful debugging output by running Cypress with this
-turned on. **Note:** you will see a LOT of messages when running with
+Cypress is built using the [debug](https://github.com/debug-js/debug) module.
+That means you can receive helpful debugging output by running Cypress with
+the environment variable `DEBUG` set to `cypress:*` prior to running `cypress run` or `cypress open`.
+**Note:** you will see a LOT of messages when running with the
```

Comment:
Just noting some facts if you want to incorporate it. It's also possible that printing a large amount of DEBUG statement could impact performance of test runs, so it should really only be used when something specifically needs debugging. It's always best to try to target the least amount of logs identify to see the problem.

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/5921/commits/d62fb2a091a634f9463813abf78913f8537e767f